modules/setup/setup.js

```diff
@@ -0,0 +1,222 @@
+                    clearInterval(progress);
+                    if (progress >= 100) {
+                        clearInterval(progressInterval);
+                        progressbar.update(100);
+                        progressbar.stop();
+                        return;
+                    }
+                    progressbar.update(progress);
+                    }, 1000);
+                    docker.modem.followProgress(dockerPullStream, (__dockerModemError, __dockerModemOutput) => {
+                        clearInterval(progress);
+                        progressbar.update(100);
+                        console.log(Helper.output('\nTezos nodes have been setup successfully on system....'));
+                        process.exit();
+                    });
+                }
+            });                     
+    }
+
+    runContainer(){
+        this.startProgressBar();
+        docker.createContainer({
+            name: `${CONTAINER_NAME}`,
+            Image: `${IMAGE_TAG}`,
+            Tty: true,
+            ExposedPorts: {
+                '18731/tchildprocess:': {}
+            },
+            PortBindings: {
+                '18731/tchildprocess': [{
+                HostPort: '18731'
+                }]
+            },
+            NetworkMode: 'host',
+            Cmd: [
+                '/bin/bash',
+                '-c',
+                'cd /usr/local/bin && start_nodes.sh && tail -f /dev/null'
+            ]
+        },
+        function(err, container) {
+            container.start({}, function(err, data) {
+                if (err) {
+                    console.log(Helper.outputError('Check whether docker is installed or not'));
+                }
+            });
+        });
+    }
+
+    startProgressBar() {
+        const _cliProgress = require('cli-progress');
+        console.log('starting the nodes.....');
+        let progress = 0;
+        let progressInterval;
+        const progressbar = new _cliProgress.Bar({
+            format: 'Progress [{bar}] {percentage}%'
+        },
+            _cliProgress.Presets.shades_classic
+        );
+
+        progressInterval = setInterval(() => {
+            progressbar.start(100, progress);
+            progress = progress + 3.6;
+            clearInterval(progress);
+            if (progress >= 100) {
+                clearInterval(progressInterval);
+                progressbar.update(100);
+                progressbar.stop();
+                return;
+            }
+            progressbar.update(progress);
+        }, 1000);
+        this.confirmNodeStatus();
+    }
+
+
+    confirmNodeStatus(){   
+        setTimeout(() => {
+            const _cliProgress = require('cli-progress');
+            const progressbar = new _cliProgress.Bar({
+                format: 'Progress [{bar}] {percentage}%'
+            },
+                _cliProgress.Presets.shades_classic
+            );
+
+            const myInterval = setInterval(() =>{
```

Comment:
variable name should not be  'myInterval', 'yourinterval', 'neighboursInterval' etc.